util/linuxfw/iptables.go

```diff
@@ -17,13 +23,61 @@ func DebugIptables(logf logger.Logf) error {
 	return nil
 }
 
+// checkBinaryExists checks if the given binary exists in the system path.
+func checkBinaryExists(bin string) bool {
+	_, err := exec.LookPath(bin)
+	return err == nil
+}
+
 // DetectIptables returns the number of iptables rules that are present in the
 // system, ignoring the default "ACCEPT" rule present in the standard iptables
 // chains.
 //
-// It only returns an error when the kernel returns an error (i.e. when a
-// syscall fails); when there are no iptables rules, it is valid for this
-// function to return 0, nil.
+// It only returns an error when there is no iptables binary, or when iptables -S
+// fails. In all other cases, it returns the number of non-default rules.
 func DetectIptables() (int, error) {
-	panic("unused")
+	// TODO(kevinliang10): Check ip6tables in case someone is running IPv6 only.
+	if !checkBinaryExists("iptables") && !checkBinaryExists("ip6tables") {
```

Comment:
The exec.Command invocations below will implicitly be doing `exec.LookPath`, do we need both?

Reply:
I guess we don't need both. I think it's easier to backtrace the error, but the error message wouldn't make too much of a change.